manpages/sources/alignak-scheduler.rst

```diff
@@ -26,16 +23,14 @@ DESCRIPTION
 
 Alignak scheduler daemon.
 
-The **alignak-scheduler** manages the dispatching of checks and actions sent to alignak-reactionner and alignak-poller based on configuration sent to it by alignak-arbiter.
+The **alignak-scheduler** manages the dispatching of checks and actions sent to
+alignak-reactionner and alignak-poller based on configuration sent to it by alignak-arbiter.
 
 OPTIONS
 =======
 
-  -c INI-CONFIG-FILE, --config=INI-CONFIG-FILE  Config file
+  -c INI-CONFIG-FILE, --config=INI-CONFIG-FILE  Daemon configuration file
   -d, --daemon                                  Run in daemon mode
   -r, --replace                                 Replace previous running scheduler
   -h, --help                                    Show this help message
-  --version                                     Show program's version number 
```

Comment:
Sorry, I did not notice this the first time. I think version still exists (comes with parser init : https://github.com/Alignak-monitoring/alignak/blob/develop/alignak/util.py#L1324)